cohortextractor/query_engines/mssql.py

```diff
@@ -162,21 +162,33 @@ def get_output_column_name(node):
         else:
             raise TypeError(f"Unhandled type: {node}")
 
-    def get_sources_from_category_definitions(self, definitions, sources=None):
-        sources = sources or set()
+    def get_query_nodes_from_category_definitions(self, definitions, query_nodes=None):
```

Comment:
Was `query_node` a bad term? I mean, I think it's better than `source`...

Reply:
I don't think it's a bad term in general, and it's definitely better than "source" in this context. I think we were just saying that `get_parent_nodes()` might be a better method name than `get_query_node_references()`. Something that makes it clear that we're just interested in walking the graph and aren't fussed about exactly what sort of objects we're getting back or what sort of role they play.

Reply:
Maybe I'll just leave it as it is for now and it can be renamed again when we do other renaming things, if we want to

Reply:
Yep, sorry I didn't mean it needed changing now. I think that's part of moving some of this logic onto the DSL objects themselves, which we can do at a later point.